connectivity/check/deployment.go

```diff
@@ -266,7 +266,7 @@ func newDaemonSet(p daemonSetParameters) *appsv1.DaemonSet {
 							ReadinessProbe:  p.ReadinessProbe,
 							SecurityContext: &corev1.SecurityContext{
 								Capabilities: &corev1.Capabilities{
-									Add: []corev1.Capability{"NET_RAW"},
+									Add: []corev1.Capability{"NET_ADMIN", "NET_RAW"},
```

Comment:
Is this proposing to grant `NET_ADMIN` rights to connectivity test deployments in users' production environments? That seems like a sort of security footprint we are explicitly trying to avoid in a "safe" tool like this. :thinking: